src/test/java/com/algorand/algosdk/transaction/TestTransaction.java

```diff
@@ -211,6 +212,21 @@ public void testAssetParamsValidation() throws Exception
             .hasMessageContaining("asset metadataHash '" +  badMetadataHash  + "' is not base64 encoded");
 
         assertThatThrownBy(() -> new AssetParams(
+                BigInteger.valueOf(100),
+                3,
+                false,
+                "tst",
+                "testcoin",
+                "websitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsitewebsite",
```

Comment:
This is significantly longer than 96 characters. Additionally, I'd recommend using something more concise and self-explanatory such as `'a'.repeat(97)`.

Reply:
good idea